### ISSUE TITLE
Authoritative repo

### ISSUE THREAD

**Issue by ojab** [HoneyryderChuck/idnx#1]
There are this repo & https://gitlab.com/honeyryderchuck/idnx, but nowhere is said where the authoritative one and where the mirror is. Since gitlab is the place for httpx, it's kinda expected to have primary idnx repo also on gitlab, but https://gitlab.com/honeyryderchuck/httpx/-/blob/master/doc/release_notes/0_15_0.md says that it's on github and currently github repo has two additional commits on top of gitlab.

TLDR: please explicitly make either gitlab repo or this one as a mirror.

**Comment by HoneyryderChuck** [comment]
Hi @ojab,

The repo at gitlab is just another dev repo, a "fork"  in a way. It historically started started there, but then I moved to github, as CI was easier to set up using github actions for this case.

The authoritative repo is this one at github. It's the one defined in the gemspec, and linked to in the idnx gem page at rubygems. IMO this should be enough to declare it the authoritative repo. Do you think I should do anything else to make it more clear?

**Comment by ojab** [comment]
I guess gitlab repo description could say that it's a mirror/playground like it says for httpx:
![image](https://user-images.githubusercontent.com/153388/124352617-082eca80-dc0a-11eb-9aa8-ad4beb301fb4.png)
It wouldn't be an issue if idnx repos had a different owner or main repo of httpx was on github, but since main httpx is on gitlab _and_ idnx [so far] seems like an utility gem for httpx, it feels that idnx also should reside on gitlab.

**Comment by HoneyryderChuck** [comment]
I've removed the gitlab project to avoid the confusion. I don't think it was necessary, but I agree that it might be confusing, giving it's the same author.

A couple of things just to clear some confusion:

* idnx is not a utility gem for idnx. It was designed to be totally independent, and although I developed it to enhance httpx, it can be used anywhere where you'd want IDNA 2008 punycode conversion.
* I'd also like it to be at gitlab. However, I couldn't setup ruby CI on multiple OSes as easily as I did for github actions. And I like to keep the repos as close to the CI as possible. I'd move it to gitlab if someone could figure out a way to port the test suite, but I won't invest any time on it myself anymore, and given gitlab's focus on becoming the devops / cloud deployment docking station, I don't believe it would pay off anyway.

Closing the issue, as it's been adressed.
